doc/holochain_101/src/links/remove_link.md

```diff
@@ -0,0 +1,17 @@
+# Remove Link
+
+`Recommended` : read link_entries documentation before reading this documentation.
+
+Remove Link is the process of marking a link as deleted in the DHT. For this process to happen, the remove_link process has to get all links that correspond with the target. This is each time a Link is commited, it is linked against the address of the link entry rather than the target. For this reason, it is why we have to run a get_links to obtain all the corresponding addresses that contain that target and set those links as deleted. This allows for links with the same target to be added after they have deleted e.g social media friend that has been removed and readded. The RemoveLink ends up adding an entry of LinkRemove to signifiy that this link has been deleted and if a similar link of LinkAdd with the same hash is added after, it will act as a tombstone and not acknowledge it.
```

Comment:
Could you please re-write this paragraph?  I find it confusing as to what's being communicated.  The main points seem to be about 1. marking for deletion, 2. the networking cost of getting other links.  3. the tombstone set.

Reply:
Absolutely! :)